### PR TITLE
[BACK-81] Tests for the methods I implemented

### PR DESCRIPTION
**Pull request by acreturs**
### Description of Issue
I had the task to implement tests for the method I coded. This includes Hackernews and OpenAI methods

### How I implemented
I used console.log for most of the methods since the answers of chatgpt are not predictable and the contribtuor count is constantly changing. I only use propper testing for the categories.
### Other
Not sure how much documentation is necessary for this

### CODE REVIEW

File: packages/graphql_server/src/supabaseUtils.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ import {
   ProjectUpdate,
   ProjectInfo
 } from '../types/supabaseUtils'
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (pczern) [inline packages/graphql_server/src/supabaseUtils.ts:13]: remove

File: packages/graphql_server/src/server.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import Fastify, { FastifyReply, FastifyRequest } from 'fastify'
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```
Comment (pczern) [inline packages/graphql_server/src/server.ts:2]: remove

**Review by pczern — APPROVED** [review]
merge only if disable comments removed. test files are in general not checked for code quality as everybody writes their own tests in their preferred style currently and tests are arbitrarily created atm. Adding the error to console.log is an improvement, but better would be to throw the error in future and catch it where we want to handle it

**Comment by thomas-woehrle** [comment]
It was decided that we won't merge the tests with main

**Pull request closed without merging**